internal/clusterinfo/types.go

```diff
@@ -115,8 +115,8 @@ func (t *TopicStats) Add(a *TopicStats) {
 	if a.Paused {
 		t.Paused = a.Paused
 	}
-	found := false
 	for _, aChannelStats := range a.Channels {
+		found := false
```

Comment:
nice fix 👍

Reply:
it only took me about a hundred printf's to find it =)